src/boxx/Resultat.java

```diff
@@ -0,0 +1,34 @@
+package boxx;
+
+public class Resultat {
+    public static void main(String[] args) {
+        BoxInterface defoltBox = new DefoltBox(3,3,3,60);
```

Comment:
pls check style (spaces for readability)
You can try to set up checkstyle for your IntellijIdea